src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj

```diff
@@ -15,11 +15,16 @@
     <CLSCompliant>false</CLSCompliant>
     <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">dotnet5.4</PackageTargetFramework>
     <UsePackageTargetRuntimeDefaults>true</UsePackageTargetRuntimeDefaults>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='net461'">true</IsPartialFacadeAssembly>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(TargetGroup)' == 'netcore50' and '$(ProjectJson)' == '' ">
     <ProjectJson>netcore50\project.json</ProjectJson>
     <ProjectLockJson>netcore50\project.lock.json</ProjectLockJson>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(TargetGroup)' == 'net461' and '$(ProjectJson)' == '' ">
```

Comment:
@bartonjs - Minor nit but at some point it is probably worth trying to combine some of these project.json files.